core/debugger/remote_debugger_peer.cpp

```diff
@@ -44,8 +44,10 @@ bool RemoteDebuggerPeerTCP::has_message() {
 
 Array RemoteDebuggerPeerTCP::get_message() {
 	MutexLock lock(mutex);
-	ERR_FAIL_COND_V(!has_message(), Array());
-	Array out = in_queue.front()->get();
+	List<Array>::Element *E = in_queue.front();
+	ERR_FAIL_NULL_V(E, Array());
```

Comment:
The previous auto-generated error would be somewhat clear (mentioning `!has_message()`) but this one would be cryptic, so worth having a proper message if it's something that can happen in the wild.
```suggestion
	ERR_FAIL_NULL_V_MSG(E, Array(), "No remote debugger messages in queue.");
```